webdriver/tests/bidi/browsing_context/navigate/wait.py

```diff
@@ -65,18 +65,18 @@ async def test_slow_page(bidi_session, new_tab, url, wait, expect_timeout):
         "/webdriver/tests/bidi/browsing_context/navigate/support/empty.html?pipe=trickle(d10)"
     )
 
-    wait_for_navigation = asyncio.wait_for(
-        bidi_session.browsing_context.navigate(
-            context=new_tab["context"], url=page_url, wait=wait
-        ),
-        timeout=1,
-    )
-
     if expect_timeout:
```

Comment:
I wonder if this should be in a helper function, given it's apparently the same in both files.

Reply:
Good point! I created a helper function and used it also in the 3rd test following a similar pattern, which I missed initially.